docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -117,6 +117,26 @@ In addition to these models, we supports experimental models for object detectio
 `Deformable_DETR <https://arxiv.org/abs/2010.04159>`_ is `DETR <https://arxiv.org/abs/2005.12872>`_ based model, and it solves slow convergence problem of DETR. `DINO <https://arxiv.org/abs/2203.03605>`_ improves Deformable DETR based methods via denoising anchor boxes. Current SOTA models for object detection are based on DINO.
 `Lite-DINO <https://arxiv.org/abs/2303.07335>`_ is efficient structure for DINO. It reduces FLOPS of transformer's encoder which takes the highest computational costs.
 
+.. note::
```

Comment:
Is this related to the 1.5.0? Or missed from previous merge-back?

Reply:
i think it was added at 1.5.0. I've checked here https://github.com/openvinotoolkit/training_extensions/compare/develop...releases/1.5.0